cosmos/dbt/graph.py

```diff
@@ -342,12 +342,13 @@ def load_from_dbt_manifest(self) -> None:
         with open(self.project.manifest_path) as fp:  # type: ignore[arg-type]
             manifest = json.load(fp)
 
-            for unique_id, node_dict in manifest.get("nodes", {}).items():
+            resources = {**manifest.get("nodes", {}), **manifest.get("sources", {}), **manifest.get("exposures", {})}
```

Comment:
It would be nice if we could try to fetch node types based on potential new resource types the user defined in the `node_converters` - we can do this in a follow up ticket

Reply:
Logged the follow up ticket in: #598 598